code/modules/mining/ore.dm

```diff
@@ -16,6 +16,31 @@
 			user << "<span class='info'>Not enough fuel to smelt [src].</span>"
 	..()
 
+/obj/item/weapon/ore/Crossed(AM as mob|obj)
+	var/obj/item/weapon/storage/bag/ore/OB
+	var/turf/simulated/floor/F = get_turf(src)
+	if(loc != F)
+		return ..()
+	if(ishuman(AM))
+		var/mob/living/carbon/human/H = AM
```

Comment:
Ah, true. I guess I should stick with the nomenclature of "human" for actual, non-xeno "humans", species be darned.